t/12_sles4sap_publicccloud.t

```diff
@@ -177,6 +177,9 @@ subtest "[stop_hana]" => sub {
             return; }
     );
     my $self = sles4sap_publiccloud->new();
+    my $mock_pc = Test::MockObject->new();
```

Comment:
Not sure which one is better but here:
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18937/files#diff-19cb97f4561f758bdb619417961e20a06b7f877f46f3ccc27be414f0ae54a3afR23
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18937/files#diff-19cb97f4561f758bdb619417961e20a06b7f877f46f3ccc27be414f0ae54a3afR118

I did it differently. I think that both implementation are achieving the same: 
- mock the wait_for_ssh, avoiding the real implementation to be invoked
- not testing anything about it: so both implementation does not later check anything about the mock execution (does anyone call it? how many time? ...)